tests/redux/reducers/award/awardReducer-test.jsx

```diff
@@ -0,0 +1,420 @@
+                        modification_number: '951',
+                        description: 'MANAGEMENT AND OPERATION OF THE OAK RIDGE NATIONAL LABORATORY - OBLIGATE $154,805,714.00 IN DOE APPROPRIATED FUNDING AND $0.00 IN NON-APPROPRIATED FUNDS; CUMULATIVE NA FUNDS - $431,254,316.85.'
+                    },
+                    {
+                        id: 36839,
+                        type: 'D',
+                        type_description: 'Definitive Contract',
+                        action_date: '10/26/2015',
+                        action_type: null,
+                        federal_action_obligation: '$498,104',
+                        modification_number: '913',
+                        description: 'MANAGEMENT AND OPERATION OF THE OAK RIDGE NATIONAL LABORATORY - TAS::68 0108::TAS - OBLIGATE SPP EPA FUNDING - OCTOBER 2015'
+                    },
+                    {
+                        id: 36757,
+                        type: 'D',
+                        type_description: 'Definitive Contract',
+                        action_date: '10/26/2015',
+                        action_type: null,
+                        federal_action_obligation: '$150,330,700',
+                        modification_number: '912',
+                        description: 'MANAGEMENT AND OPERATION OF THE OAK RIDGE NATIONAL LABORATORY - OBLIGATE $150,330,700.26 IN DOE APPROPRIATED FUNDING; NON-APPROPRIATED FUNDS CURRENT ACTION = $1,523,053.45; NON-APPROPRIATED FUNDS CUMULATIVE SINCE MOD. 234 = $413,663,188.76.'
+                    },
+                    {
+                        id: 36791,
+                        type: 'D',
+                        type_description: 'Definitive Contract',
+                        action_date: '10/15/2015',
+                        action_type: null,
+                        federal_action_obligation: '$0',
+                        modification_number: '911',
+                        description: 'MANAGEMENT AND OPERATION OF THE OAK RIDGE NATIONAL LABORATORY - REVISE APPENDIX E TO INCORPORATE RCN OR-62'
+                    },
+                    {
+                        id: 34869,
+                        type: 'D',
+                        type_description: 'Definitive Contract',
+                        action_date: '10/30/2014',
+                        action_type: null,
+                        federal_action_obligation: '-$14,274',
+                        modification_number: '860',
+                        description: 'MANAGEMENT AND OPERATION OF THE OAK RIDGE NATIONAL LABORATORY - RECOVERY TAS::89 0227::TAS - DEOBLIGATE SC ARRA FUNDING FROM PROJECT ENTITLED "NUCLEAR PHYSICS - NUCLEAR SCIENCE WORKFORCE"'
+                    }
+                ]
+            };
+            const updatedState = awardReducer(initialState, action);
+            // the value should be equal
+            expect(updatedState.transactions).toEqual(action.transactions);
```

Comment:
The initial state is an empty array, so appending data to an empty array doesn't give us assurance that this Redux action is actually appending and not replacing.